src/steth/BaseV2Vault.sol

```diff
@@ -150,6 +150,8 @@ abstract contract BaseV2Vault is sc4626, IFlashLoanRecipient {
 
     function _multiCall(bytes[] memory _callData) internal {
         for (uint256 i = 0; i < _callData.length; i++) {
+            if (_callData[i].length == 0) continue;
```

Comment:
I dont see a scenario where the backend might send a empty callData unless in case of an error and in such a case I would say its better to have the call reverted so that we can debug the issue.

Reply:
you have a point there but I don't see it as an issue. We can `revert` instead of `continue` but that would complicate things in `_withdrawToVault` func in scWETHv2.sol. There we initialize a bytes array for multicall which might not have every item set if we are not going to withdraw from every protocol supported. Handing that case would require additional changes which I consider unnecessary. That said this pr might not have much value

Reply:
I see your point.